airflow/operators/python.py

```diff
@@ -513,9 +513,9 @@ class PythonVirtualenvOperator(_BasePythonVirtualenvOperator):
         "requirements file" as specified by pip.
     :param python_version: The Python version to run the virtual environment with. Note that
         both 2 and 2.7 are acceptable forms.
-    :param use_dill: Whether to use dill to serialize
-        the args and result (pickle is default). This allow more complex types
-        but requires you to include dill in your requirements.
```

Comment:
If this parameter is declared in the base class `_BasePythonVirtualenvOperator`, and the derived classes only pass it back to the super(), then I think we could just remove it from docstrings and __ini__() method in all derived classes in order to avoid a code duplication.